packages/medical-domain/test/repositories/statistics/insulin-statistics.service.test.ts

```diff
@@ -55,9 +81,9 @@ describe('getBasalBolusData', () => {
 
     const basalBolusData = BasalBolusStatisticsService.getBasalBolusData(basals, bolus, 2, dateFilterTwoWeeks)
     const expectBasalBolusData = {
-      basal: 1,
+      basal: 1.25,
```

Comment:
So, there was a bug before?

Reply:
No, it's just that I've added an extra dataset that I need in another test, so I need to change the output value.